### ISSUE TITLE
Debian stretch is not supported

### ISSUE THREAD

**Issue by Magomogo** [saltstack-formulas/postgres-formula#153]
```
[INFO    ] Running state [postgresql-repo] at time 07:09:13.730058
[INFO    ] Executing state pkgrepo.managed for postgresql-repo
[INFO    ] Executing command ['apt-key', 'add', '/var/cache/salt/minion/extrn_files/project/www.postgresql.org/media/keys/ACCC4CF8.asc'] in directory '/root'
[INFO    ] Executing command ['apt-get', '-q', 'update'] in directory '/root'
[ERROR   ] Command '['apt-get', '-q', 'update']' failed with return code: 100
[ERROR   ] stderr: E: Malformed line 1 in source list /etc/apt/sources.list.d/pgdg.list (type)
E: The list of sources could not be read.
[ERROR   ] retcode: 100
[ERROR   ] Failed to configure repo 'postgresql-repo': E: Malformed line 1 in source list /etc/apt/sources.list.d/pgdg.list (type)
E: The list of sources could not be read.
[INFO    ] Completed state [postgresql-repo] at time 07:09:19.287936 duration_in_ms=5559.368
```

**Comment by vutny** [comment]
Hi @Magomogo.

This should be resolved with PR #154 merged in few minutes ago. Thanks for the report and please test.

**Comment by javierbertoli** [comment]
Fixed on #154
